sigs.yaml

```diff
@@ -3526,6 +3526,34 @@ sigs:
     - https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-dev-tools/master/OWNERS
     - https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/OWNERS
 workinggroups:
+- dir: wg-ai-conformance
+  name: AI Conformance
+  mission_statement: "The goal of this group is to define a standardized set of capabilities,
+    APIs, and configurations that a Kubernetes cluster must offer to reliably and
+    efficiently run AI/ML workloads. \n"
+  charter_link: charter.md
+  stakeholder_sigs:
+  - Architecture
```

Comment:
I believe this should list the same SIGs that are listed as stakeholders in your charter.

Reply:
Okay, so does everybody agree to list only: SIG Architecture and SIG Testing for now? Please let me know so I can make the change and get this merged asap to start meeting

Reply:
+1 as SIG Testing TL.